roaster-bom/pom.xml

```diff
@@ -168,6 +180,48 @@
 				<version>${version.com.oracle.ojdbc}</version>
 			</dependency>
 
+			<!-- Selenide -->
+			<dependency>
+				<groupId>com.codeborne</groupId>
+				<artifactId>selenide</artifactId>
+				<version>${version.codeborn.selenide}</version>
+				<exclusions>
+					<exclusion>
```

Comment:
ez miert kell exclude-ra ha megis be van kotve alatta?

Reply:
Régebbi verzióból másoltam, ahol eltértek a verziók túlzottan is, de javították. Kiveszem.